src/Schema/ColumnDiff.php

```diff
@@ -127,9 +127,20 @@ public function hasCommentChanged(): bool
 
     public function hasPlatformOptionsChanged(): bool
     {
-        return $this->hasPropertyChanged(static function (Column $column): array {
-            return $column->getPlatformOptions();
-        });
+        foreach (
+            [
+                static fn (Column $column): ?string => $column->getCharset(),
+                static fn (Column $column): ?string => $column->getCollation(),
+                static fn (Column $column): mixed => $column->getMinimumValue(),
+                static fn (Column $column): mixed => $column->getMaximumValue(),
+            ] as $property
```

Comment:
Since more property exists in the PlatformOptions, like, at least, enumType this doesn't give the same behavior (cf https://github.com/doctrine/dbal/pull/7238)

Before when changing the enumType this method returned true ; now it returns false. (and maybe there are others important platform options)

Not sure of the impact